### PR TITLE
add contact-info section html & css

### LINKED ISSUE #9: Contact-info
## 6. contact info

> assigned to Boam

__As a site visitor, I want to know how I can contact each of them__

### Repository
-  [x] developed on a branch called contact-info
### HTML
- [x] An section element ...
- [x] ... with the author's contact info
- [x] ... and class "contact-info"
### CSS 
- [x] the "contact-info" class

### PR DESCRIPTION
**Pull request by Boam1994**
I completed the contact-info section. Could you review it?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,6 +12,30 @@
 </head>
 
 <body>
+  <!--Contact info-->
+  <section class="contact-info">
+      <div class="center">
+        <h1>Get In Touch</h1>
```
Comment (adekimpianna) [inline index.html]: hi boam, `h1` is only used on the top of the page, can you please change it to `h3`?  we are only going to make one page and contact info will be at the bottom before `footer`. sorry i did not mentioned it clearly before

**Comment by harunaltunhr** [comment]
You need to link your pull request to an issue. I did it this time, but you can do it next time writing in comment "#closes issue number"

**Comment by adekimpianna** [comment]
@Boam1994 it looks great!! but can you still change the `h1` and use `h2` or `h3` please 🙏 
image

**Comment by Feruzteame** [comment]
hey @Boam1994  which part is this ? couse am working with section. we can not work in the same box. can you explain please?

**Comment by adekimpianna** [comment]
> 
> 
> hey @Boam1994 which part is this ? couse am working with section. we can not work in the same box. can you explain please?

@Feruzteame I use the old wireframe from HFY, over there there is only one section. But we are working with many people, so I added extra sections. Your section is introduction section **it will be on the top of main area** and @Boam1994 is making contact section, **it will be at the bottom of main area** They are not the same 😄

**Comment by Feruzteame** [comment]
I was shocked. I said "who touch my section" haha. Thank you boam. Understood !

**Review by miroslavveljanoski — APPROVED** [review]
This seems great to me. Just change the h1 to h2 as @adekimpianna mention, so we can merge this branch to the master

**Comment by miroslavveljanoski** [comment]
@Boam1994 Also you should resolve the conflicts, otherwise we cannot merge your branch

**Review by harunaltunhr — APPROVED** [review]
Let's do it :+1:

**Comment by Feruzteame** [comment]
just resolve the conflict. it looks great

**Comment by Boam1994** [comment]
Sorry guys being late. I'm going to fix and try to resolve conflicts. I will ask for help if needed.

**Comment by adekimpianna** [comment]
> 
> 
> Sorry guys being late. I'm going to fix and try to resolve conflicts. I will ask for help if needed.

@Boam1994 maybe we can solve the conflict tomorrow at 12.00 when we have a video call with stephane?

**Pull request merged**